samples/Main.js

```diff
@@ -11,6 +11,9 @@ function Main() {
       <BasicComponents />
       <h2>Recents Components</h2>
       <RecentsComponents />
+      <h2>Base Elements that should not be modified by momentum css</h2>
```

Comment:
Added these elements to the sample page to verify they weren't getting style overrides from momentum.